src/cursor/cur_backup.c

```diff
@@ -207,22 +351,204 @@ __backup_log_append(WT_SESSION_IMPL *session, WT_CURSOR_BACKUP *cb, bool active)
     return (ret);
 }
 
+/*
+ * __backup_config --
```

Comment:
This is another routine where it makes sense to me to separate the dup/non-dup cases, they really don't share anything, do they?

Reply:
I'll consider this and see how it turns out. I generally like having a single configuration parsing function rather than two because we do want to error so we'd have to check the set anyway. And then if the API changes, it is easier to remember and see we only need to change one location.

Reply:
In general, WiredTiger does not check for configurations that are not applicable to the current API, so by doing this, you're diverging a bit from the rest of the code. I get that it's more likely users will mistakenly use backup configurations with incremental backup configurations, but I'm not sure if that's a compelling argument.

It will be easier to understand/remember if there's a configuration function per API than if there's a configuration function that covers multiple distinct APIs.